WordPress/Classes/ViewRelated/Blog/Settings/SiteSettingsViewModel.swift

```diff
@@ -0,0 +1,52 @@
+import Foundation
+import SwiftUI
+import Combine
+
+final class SiteSettingsViewModel: ObservableObject {
+    private let blog: Blog
+    private let service: BlogService
+
+    let onDismissableError = PassthroughSubject<String, Never>()
+
+    init(blog: Blog) {
```

Comment:
Wdyt of injecting the service?
```suggestion
    init(blog: Blog, service: BlogService = BlogService(coreDataStack: ContextManager.shared)) {
```

Reply:
Good point – gotta keep is testable. Updated.

I'm prioritizing updating the UI. The original implementation had no tests, but I'll try to add some (and make it more testable).